Update Game Instructions (#2)
ninanikitina/reviewing-a-pull-request: README.md

You can play the game at: https://githubschool.github.io/github-games/

### Instructions for playing the game

> 1. Press the space bar to begin.

Reviewer: comment for 3rd task

> 2. Use the up and down arrow keys to rotate the shape.

Reviewer: another comment for the third task

> 
> 1. Press the space bar to begin.
> 2. Use the up and down arrow keys to rotate the shape.
> 3. Use the left and right arrow keys to position the shape.
> 4. The goal is to create complete rows with no empty spaces.
> 5. When completed, the rows will disappear.

Reviewer: Test comments.
```suggestion

1. Press a space bar to begin.
2. Use the up and down keys to rotate the shape.
3. Use the left and right keys to position the shape.
4. The goal is to create complete rows without empty spaces.
5. When completed, the rows will disappear.
```
